### ISSUE TITLE
UNQUOTED_STRUCT_KEY only reported for implicit declaration

### ISSUE THREAD

**Issue by KamasamaK** [cflint/CFLint#405]
This rule is not triggered when using dot notation to declare a new key.

Reported:
```
<cfset sampleStruct = {
    aKey="aValue",
    anotherKey="anotherValue"
}>
```

Unreported:
`<cfset sampleStruct.yetAnotherKey = "yetAnotherValue">`

It should be consistent and report both, recommending using array-notation when declaring new keys on a struct.
